tests/fixtures/explore.ts

```diff
@@ -1,8 +1,13 @@
-import type { Page, Locator } from '@playwright/test';
```

Comment:
We should prettier --write this file.

Reply:
I'll do you one better, now we automatically run prettier and eslint when test files are updated!

Reply:
💯